test/fixtures/restaurants.yml

```diff
@@ -0,0 +1,17 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
```

Comment:
we use factories (using `FactoryGirl`) over fixtures as they provide more flexability